proto/xudp2p.proto

```diff
@@ -31,7 +31,7 @@ message NodeState {
     map<string, string> lnd_pub_keys = 6;
     map<string, string> token_identifiers = 7;
     map<string, LndUris> lnd_uris = 8;
-    string connext_address = 9;
+    string connext_identifier = 9;
```

Comment:
Just a thought, but going forward I think it would make sense to keep this name even *if* for some reason connext renames this field again. "Identifier" is generic and we can use that for our p2p layer regardless of what connext calls it.